sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -223,12 +221,12 @@ def list_configuration_settings(
     @distributed_trace
     def get_configuration_setting(
         self,
-        key,
-        label=None,
-        etag="*",
-        match_condition=MatchConditions.Unconditionally,
-        **kwargs
-    ):  # type: (str, Optional[str], Optional[str], Optional[MatchConditions], **Any) -> ConfigurationSetting
+        key,  # type: str
+        label=None,  # type: Optional[str]
+        etag="*",  # type: Optional[str]
+        match_condition=MatchConditions.Unconditionally,  # type: Optional[MatchConditions]
+        **kwargs  # type: Any
+    ):  # type: (...) -> Union[None, ConfigurationSetting]
```

Comment:
This still seems odd to me, but if it's the expected behavior then I think we should probably update the `:rtype:` in the docstring and perhaps explain in `:return:` what case `None` could be returned

Reply:
Ah I didn't see the auto-merge tag, sorry